src/vcfio/utils/str_utils.py

```diff
@@ -33,12 +31,46 @@ def standardize_chromosome(raw_chr):
     return chromosome
 
 
-def calculate_zygosity(gt: AnyStr) -> Zygosity:
+def calculate_zygosity(genotype: str) -> Optional[Zygosity]:
```

Comment:
python typing convention is to be as broad as possible for arguments and as accurate as possible for return values. The change of AnyStr to str is against

Reply:
This function is private to vcfio and not part of the API, so it is guaranteed to be called with type 'str'.
In this case, AnyStr is just the wrong type and it is flagged by the type checker.